public/docs/css/style.css

```diff
@@ -1 +1 @@
-top:2em;border-top:1px solid #ddd;background-image:linear-gradient(180deg,#fff,#f9f9f9)}.content div:first-child+h1,.content h1:first-child{border-top-width:0;margin-top:0}.content h2{font-size:20px;margin-top:4em;margin-bottom:0;border-top:1px solid #ccc;padding-top:1.2em;padding-bottom:1.2em;background-image:linear-gradient(180deg,hsla(0,0%,100%,.4),hsla(0,0%,100%,0))}.content h1+div+h2,.content h1+h2{margin-top:-21px;border-top:none}.content h3,.content h4,.content h5,.content h6{font-size:15px;margin-top:2.5em;margin-bottom:.8em}.content h4,.content h5,.content h6{font-size:10px}.content hr{margin:2em 0;border-top:2px solid #393939;border-bottom:2px solid #eaf2f6}.content table{margin-bottom:1em;overflow:auto}.content table td,.content table th{text-align:left;vertical-align:top;line-height:1.6}.content table th{padding:5px 10px;border-bottom:1px solid #ccc;vertical-align:bottom}.content table td{padding:10px}.content table tr:last-child{border-bottom:1px solid #ccc}.content table tr:nth-child(odd)>td{background-color:#ebf3f6}.content table tr:nth-child(even)>td{background-color:#ebf2f6}.content dt{font-weight:700}.content dd{margin-left:15px}.content dd,.content dt,.content li,.content p{line-height:1.6;margin-top:0}.content img{max-width:100%}.content code{background-color:rgba(0,0,0,.05);padding:3px;border-radius:3px}.content pre>code{background-color:transparent;padding:0}.content aside{padding-top:1em;padding-bottom:1em;text-shadow:0 1px 0 #a0c6da;margin-top:1.5em;margin-bottom:1.5em;background:#8fbcd4;line-height:1.6}.content aside.warning{background-color:#c97a7e;text-shadow:0 1px 0 #d18e91}.content aside.success{background-color:#6ac174;text-shadow:0 1px 0 #80ca89}.content aside:before{vertical-align:middle;padding-right:.5em;font-size:14px}.content .search-highlight{padding:2px;margin:-2px;border-radius:4px;border:1px solid #f7e633;text-shadow:1px 1px 0 #666;background:linear-gradient(to top left,#f7e633,#f1d32f)}.content blockquote,.content pre{background-color:#292929;color:#fff;padding:2em 28px;margin:0;width:50%;float:right;clear:right;box-sizing:border-box;text-shadow:0 1px 2px rgba(0,0,0,.4)}.content blockquote>p,.content pre>p{margin:0}.content blockquote a,.content pre a{color:#fff;text-decoration:none;border-bottom:1px dashed #ccc}.content blockquote>p{background-color:#1c1c1c;border-radius:5px;padding:13px;color:#ccc;border-top:1px solid #000;border-bottom:1px solid #404040}@media (max-width:930px){.tocify-wrapper{left:-230px}.tocify-wrapper.open{left:0}.page-wrapper{margin-left:0}#nav-button{display:block}.tocify-wrapper .tocify-item>a{padding-top:.3em;padding-bottom:.3em}}@media (max-width:700px){.dark-box{display:none}.tocify-wrapper .lang-selector{display:block}.page-wrapper .lang-selector{display:none}.content aside,.content dl,.content h1,.content h2,.content h3,.content h4,.content h5,.content h6,.content ol,.content p,.content table,.content ul{margin-right:0}.content blockquote,.content pre{float:none;width:auto}}
```

Comment:
Get updated CSS from @basit-tkxel and include it there. Add that code manually instead of regenerating the api doc perhaps.

Reply:
Added the css from Basit in "public/docs/css/style.css".